test/cli/test-other.py

```diff
@@ -825,3 +825,25 @@ def test_file_duplicate_2(tmpdir):
         '3/3 files checked 0% done'
     ]
     assert stderr == ''
+
+
+def test_cgf_with_relative_path(tmpdir):
+
+    test_file = os.path.join(tmpdir, 'test.c')
+    with open(test_file, 'wt'):
+        pass
+
+    test_cfg = os.path.join(tmpdir, '../../cppcheck.cfg')
```

Comment:
hmm.. the intention is to put cppcheck.cfg in the same folder as the cppcheck binary. It's not located relative to the `tmpdir`. Maybe you can rename and use the `__lookup_cppcheck_exe` function to determine where you should put cppcheck.cfg.